packages/engine/Source/Shaders/Voxels/convertUvToEllipsoid.glsl

```diff
@@ -104,7 +101,7 @@ vec3 convertUvToShapeUvSpace(in vec3 positionUv) {
     #endif
 
     // Compute height
-    #if defined(ELLIPSOID_HAS_SHAPE_BOUNDS_HEIGHT_FLAT) || defined(ELLIPSOID_HAS_RENDER_BOUNDS_HEIGHT_FLAT)
+    #if defined(ELLIPSOID_HAS_SHAPE_BOUNDS_HEIGHT_FLAT)
         // TODO: This breaks down when minBounds == maxBounds. To fix it, this
         // function would have to know if ray is intersecting the front or back of the shape
         // and set the shape space position to 1 (front) or 0 (back) accordingly.
```

Comment:
Can this comment be deleted now?

Reply:
Unfortunately no, that's a separate issue: https://github.com/CesiumGS/cesium/issues/11017

Reply:
OK that makes sense, thanks!